Exam_11.js

```diff
@@ -0,0 +1,174 @@
+// Bài 1
+// Viết 1 hàm trả về 1 array, nhận vào 2 tham số, tham số thứ 1 là ký tự cần lặp, tham số thứ 2 là số lần lặp
+// vd: đầu vào ('ahihi', 3)
+// Thì Kết quả là:
+// ['ahihi', 'ahihi', 'ahihi']
+// Yêu cầu, viết ít nhất 2 cách khác nhau
+//method 1
+const repeatingCharacters = (a = '', n) => {
+  const result = []
+  for (let i = 0; i < n; i++) {
+    result.push(a)
+  }
+  return result
+}
+
+//method 2
+const repeatingCharacters_2 = (a = '', n) => {
+  const result = []
+  let i = 0
+  while (i < n) {
+    result.push(a)
+    i++
+  }
+  return result
+}
+
+//Bài 2
```

Comment:
```suggestion
// Bài 2
```
Và không được dùng tiếng Việt
Đã nhắc ở PR trước:
Tất cả mọi thứ đều phải dùng tiếng anh ngoại trừ được yêu cầu dùng ngôn ngữ. khác, hoặc bắt buộc phải dùng ngôn ngữ khác